resources/liquibase/db.changelog.xml

```diff
@@ -0,0 +1,84 @@
+               primaryKey="true"
+               primaryKeyName="users_pkey"
+            />
+         </column>
+         <column name="email" type="VARCHAR(255)">
+            <constraints nullable="false" unique="true" uniqueConstraintName="users_email_key"/>
+        </column>
+        <column name="password" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="role" type="VARCHAR(10)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="firstname" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="lastname" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="gender" type="VARCHAR(10)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="dob" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValueBoolean="false" name="isActive" type="BOOLEAN"/>
+            <column defaultValueBoolean="false" name="isVerified" type="BOOLEAN"/>
+            <column name="createdAt" type="TIMESTAMP WITH TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="updatedAt" type="TIMESTAMP WITH TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+     </createTable>
+   </changeSet>
+   <changeSet author="moroti" id="202302021026">
+        <createTable tableName="books">
+            <column autoIncrement="true" name="id" type="INTEGER">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="books_pkey"/>
+            </column>
+            <column name="title" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="isbn" type="VARCHAR(10)">
+                <constraints nullable="false" unique="true" uniqueConstraintName="books_isbn_key"/>
+            </column>
+            <column name="author" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValueBoolean="true" name="isAvailable" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
+            <column name="createdAt" type="TIMESTAMP WITH TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="updatedAt" type="TIMESTAMP WITH TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+             <column name="createdBy" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+             <column name="updatedBy" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+             <column name="totalNumber" type="INTEGER">
```

Comment:
I was expecting you to create another changeset and not delete your database. Don't do this for our next task. If you need to add a new column or remove column from an existing table, then create a new changeset.

You are already making the essence of liquibase useless in your code if you have to do this. What you did here will mess up your production if you already deployed to production and especially if you already have data.